### ISSUE TITLE
[macOS 15] Glance not working properly?

### ISSUE THREAD

**Issue by vinhquach** [chamburr/glance#49]
Has anyone else experienced some Quick Look extensions (including Glance's) malfunctioning since macOS Sequoia?

**Comment by dserodio** [comment]
Yes, qlZipInfo's author found the [reason](https://github.com/srirangav/qlZipInfo/pull/2#issuecomment-2504752813)

**Comment by chamburr** [comment]
> Yes, qlZipInfo's author found the [reason](https://github.com/srirangav/qlZipInfo/pull/2#issuecomment-2504752813)

That looks irrelevant, because we are already using the "new method". I haven't personally experienced any malfunctioning with Glance on macOS Sequoia, and I would appreciate more details if this is still a persistent issue.
